_data/tool_and_resource_list.yml

```
@@ -2879,10 +2893,15 @@
 - description: NIH research program with diverse patient health, genetic, and lifestyle data.
   id: all-of-us
   name: All of Us
+  registry:
+    tess: All of Us
   url: https://www.researchallofus.org/
 - description: Large-scale biomedical database with genetic, clinical, and lifestyle data from 500,000 UK participants.
   id: uk-biobank
   name: UK Biobank
+  registry:
+    biotools: uk-biobank
+    tess: UK Biobank
   url: https://www.ukbiobank.ac.uk/
 - description: Database of drug sensitivity and genomic data from ~1000 cancer cell lines.
```

```suggestion
- description: Database of drug sensitivity and genomic data from cancer cell lines.
```

Not sustainable to keep track of the number